README.org

```diff
@@ -50,3 +62,30 @@ installed using =brew=. The same works with =cask= and =brewfile=. For example:
 
 - cask: [colorpicker, vlc]
 #+END_SRC
+
+Defaults also can control stdin, stdout and stderr (disabling with false, enabling with true),
+see below for an example
+
+#+BEGIN_SRC yaml
+- defaults:
+    - brewfile:
```

Comment:
The way it's written, I would think that these configurations for `stdin`, `stdout` and `stderr` are applied only to `brewfile` directive. But as far as I understand from your code, it is applied to all commands because `_defaults` is a mutable variable (read other comments). Doesn't sound cool to me. As suggestion, this could be renamed to `dotbot-brew` to clearly indicate that this section configures behaviour of `dotbot-brew` as a whole and not just one of the directives it provides. Or defaults should passed as arguments (see my other comments).